arch/rv32i/src/lib.rs

```diff
@@ -188,20 +176,7 @@ pub extern "C" fn _start_trap() {
 
 #[cfg(all(target_arch = "riscv32", target_os = "none"))]
 extern "C" {
-    /// This is the trap handler function. This code is called on all traps,
-    /// including interrupts, exceptions, and system calls from applications.
-    ///
-    /// Tock uses only the single trap handler, and does not use any vectored
-    /// interrupts or other exception handling. The trap handler has to determine
-    /// why the trap handler was called, and respond accordingly. Generally, there
-    /// are two reasons the trap handler gets called: an interrupt occurred or an
-    /// application called a syscall.
-    ///
-    /// In the case of an interrupt while the kernel was executing we only need to
-    /// save the kernel registers and then run whatever interrupt handling code we
-    /// need to. If the trap happens while and application was executing, we have to
-    /// save the application state and then resume the `switch_to()` function to
-    /// correctly return back to the kernel.
+    /// TODO
```

Comment:
What is the TODO here? Is this a comment to yourself to write documentation before converting from a draft to non-draft PR?